docker-compose-local.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+  postgres:
+    image: postgres:16
+    container_name: postgres_db
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgrest_knack_services
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./init_scripts/knack_services:/docker-entrypoint-initdb.d
```

Comment:
I'm seeing the `.d` on this folder, and it's making me think that we should probably be able to put multiple files in here. `.d` for "drop in" folder, where files will be processed in alphabetical order. Just a thing to keep in the back of our minds.

Reply:
Yep, that's right. And I believe you can also stash a shell script here. Which is what we'd need to do bc of our need to create/operate on multiple DBs.